.ci/ansible_tests.py

```diff
@@ -66,8 +66,6 @@ def pause_if_interactive():
         run("sudo apt-get update")
         run("sudo apt-get install -y sshpass")
 
-    run("bash -c 'sudo ln -vfs /usr/lib/python2.7/plat-x86_64-linux-gnu/_sysconfigdata_nd.py /usr/lib/python2.7 || true'")
```

Comment:
Future note to see if this will cause problems in tests, once they run again

Reply:
Oh, I hadn't noticed the tests failed to start.

Reply:
Tests have succeeded